include/net/net_event.h

```diff
@@ -67,6 +67,8 @@ enum net_event_ipv6_cmd {
 	NET_EVENT_IPV6_CMD_DAD_FAILED,
 	NET_EVENT_IPV6_CMD_NBR_ADD,
 	NET_EVENT_IPV6_CMD_NBR_DEL,
+	NET_EVENT_IPV6_CMD_READY,
+	NET_EVENT_IPV6_CMD_DISCONNECTED,
```

Comment:
I'm a little puzzled on which applications need these events, as the motivation behind this patch set is not written anywhere. Are the events supposed to be sent when an IP address appears and disappears? Or is this just signalling connectivity for a particular IP version? What if an IPv6 address expires while the interface has other usable addresses, is an event supposed to be sent at that point also?